controlplane/src/core/repositories/analytics/AnalyticsRequestViewRepository.ts

```diff
@@ -721,6 +731,8 @@ export class AnalyticsRequestViewRepository {
 
     whereSql += scopedSql;
 
+    this.escapeStrings(coercedQueryParams);
```

Comment:
There could be other places too right? Ideally, we can fix this on the driver level but since we only pass strings this seems to be impossible. We should think about a better abstraction in the future.

Reply:
Fixed it in other places too